src/megatron/bridge/models/qwen_vl/modelling_qwen3_vl/transformer_block.py

```
@@ -43,6 +48,392 @@
+                sharded_prefix = deepstack_prefix
+                sharded_pp_offset = [(0, global_layer_offset, len_deepstack)]  # PP sharding offset for ShardedTensors
+            layer_sharded_state_dict = layer.sharded_state_dict(state_dict_prefix, sharded_pp_offset, metadata)
+            replace_prefix_for_sharding(layer_sharded_state_dict, state_dict_prefix, sharded_prefix)
+
+            sharded_state_dict.update(layer_sharded_state_dict)
+
+        # Add modules other than self.layers
+        for name, module in self.named_children():
+            if (not module is self.layers) and (not module is self.deepstack_merger_list):
+                sharded_state_dict.update(
+                    sharded_state_dict_default(module, f"{prefix}{name}.", sharded_offsets, metadata)
```

Flake8 (7.3.0)

[error] undefined name 'sharded_state_dict_default'

(F821)